ansible/roles/satellite-configuration/tasks/02_repository.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+- name: Fetching enabled Repos Name list from subscription of {{ORG}}
+  shell: hammer --output cve  repository list --organization "{{ORG}}" |grep Name
+  register: repo_name
+  ignore_errors: yes
+           
+- name: Creating fact from enabled Repos Name list from subscription of {{ORG}} 
+  set_fact:
+    repos_added: "{{ repos_added + [ item.split(':')[1].lstrip(' ') ] }}"
+  loop: "{{ repo_name.stdout_lines}}"
```

Comment:
What happen to this task and following tasks when `Fetching enabled Repos Name list from subscription of {{ORG}}` fails. (asking because `ignore_errors: yes` is set there)

Reply:
Fixed